packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -80,41 +171,41 @@ export const styles = (theme) => ({
     },
     '&[class*="MuiOutlinedInput-root"]': {
       padding: 9,
-      '$hasPopupIcon &, $hasClearIcon &': {
+      [`.${autocompleteClasses.hasPopupIcon} &, .${autocompleteClasses.hasClearIcon} &`]: {
         paddingRight: 26 + 4 + 9,
       },
-      '$hasPopupIcon$hasClearIcon &': {
+      [`.${autocompleteClasses.hasPopupIcon}.${autocompleteClasses.hasClearIcon} &`]: {
         paddingRight: 52 + 4 + 9,
       },
-      '& $input': {
+      [`& .${autocompleteClasses.input}`]: {
```

Comment:
These nested `$input` selectors should not change. They are directly related to OutlinedInput